pyramid_hypernova/request.py

```diff
@@ -13,8 +13,11 @@ def create_jobs_payload(jobs):
 
 
 class HypernovaQueryError(Exception):
-    def __init__(self, child_error):
+    # error_data (obj, optional): { name, message, stack }
+    def __init__(self, child_error, error_data=None):
         super().__init__(str(child_error))
+        if error_data:
+            self.error_data = error_data
```

Comment:
another approach could be to _always_ set `error_data`, but have it sometimes be `None`

that could be another way to make this a bit more robust, so that we're doing less checks overall